digitalocean/machine-mynode.yaml.tmpl

```diff
@@ -0,0 +1,17 @@
+---
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys: ${ssh_keys}
+storage:
+  files:
+    - path: /home/core/works
+      filesystem: root
+      mode: 0755
+      contents:
+        inline: |
+          #!/bin/bash
+          set -euo pipefail
+          hostname="$(hostname)"
+          echo My name is ${name} and the hostname is $${hostname}
```

Comment:
What's the point of this script?  How does "${name}" get filled?

Reply:
The script demonstrates using the Terraform template language with Terraform variables and shell variables, `${name}` is the Terraform substitution from
```
  vars = {
   ssh_keys = jsonencode(var.ssh_keys)
    name     = each.key
  }
```
while `$${hostname}` is the quoted `${hostname}` – I think this is the main thing to be aware of for when using Container Linux Configs with Terraform, so it's good to have this covered in the example.

Reply:
I see, please add a comment about this, then:
# This script demonstrates how templating and variable substitution works when using Container Linux Configs.

Reply:
Thanks, done